src/Model/Store/SessionProgressStore.php

```diff
@@ -0,0 +1,84 @@
+<?php
+
+namespace CodeCraft\Pathfinder\Model\Store;
+
+
```

Comment:
Tidy Kiwi: Remove empty line. If this is unintentional

Reply:
These _were_ intentional, but you changed my mind 😅